environment_minimum.yml

```diff
@@ -3,7 +3,6 @@ channels:
 - !!python/unicode
   'defaults'
 dependencies:
-  - Cython
```

Comment:
We cannot remove this line because this repository does not contain `*.c` files. Without this line, Travis CI fails.